components/PlantForm.js

```diff
@@ -168,14 +193,16 @@ export default function PlantForm({ onCreatePlant }) {
                 id={option.id}
                 name="fertiliserSeason"
                 value={option.value}
+                defaultChecked={isEditMode ? initialData?.fertiliserSeason.includes(option.value) : null}
               />
               <label htmlFor={option.id}>{option.label}</label>
             </div>
           ))}
         </section>
 
         <div className="button">
-          <StyledButton type="submit">Create</StyledButton>
+          <StyledButton type="submit">{isEditMode ? "Save"  : "Create"}</StyledButton>
+          <StyledButton onClick={handleCancel}>Cancel</StyledButton>
```

Comment:
- Set `type="button"` to prevent form submission on cancel click

```suggestion
          <StyledButton  type="button" onClick={handleCancel}>Cancel</StyledButton>
```